rust_src/src/lisp.rs

```diff
@@ -279,6 +287,27 @@ impl From<LispObject> for Option<LispSubrRef> {
     }
 }
 
```

Comment:
Should this be in symbols.rs?

Reply:
I don't think so. Original `specbinding` struct defined is `lisp.h`, so I think it makes more sense if we place `SpecbindingRef` in `lisp.rs` as well.